### PR TITLE
#5 Create TodoFormView class to handle add to-do

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- #5

### CODE REVIEW

File: practices/practice-one/src/scripts/views/todo-form.view.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    init() {
+        this.addEventForm()
+    }
+
+    private addEventForm() {
+        this.formElement.addEventListener('submit', (e) => {
+            e.preventDefault();
+
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/scripts/views/todo-form.view.ts]: Space

File: practices/practice-one/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import TodoFormView from "./scripts/views/todo-form.view";
+const todoFormView = new TodoFormView()
+
+if (document.readyState === 'loading') {
+    document.addEventListener('DOMContentLoaded', todoFormView.init)
+}
+else {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.ts]: Look like your config not work correctly, could you re-check it?

Comment (DuongPham-dev) [reply practices/practice-one/src/index.ts]: - I still use it for other projects. I see it still works;
- Can you tell me more?

Comment (cunguyen-agilityio) [reply practices/practice-one/src/index.ts]: Please in condition https://github.com/airbnb/javascript#blocks

File: practices/practice-one/src/index.html [hunk 3]
```diff
@@ -75,6 +77,7 @@ <h1 class="heading">Todo list with TypeScript</h1>
             <button class="btn btn-delete"></button>
         </div>
     </section>
+    <script src="./index.ts" type="module"></script>
 </body>
 
 </html>
```
Comment (DuongPham-dev) [inline practices/practice-one/src/index.html:83]: - Comment
-

**Pull request merged**